### ISSUE TITLE
Upgrade to flutter 3.22

### ISSUE THREAD

**Issue by NashIlli** [ponnamkarthik/FlutterToast#508]
Hello! Im trying to compile for Wasm with the new Flutter version. 

Im getting the following error:

```
../../../.pub-cache/hosted/pub.dev/fluttertoast-8.2.4/lib/fluttertoast_web.dart:2:8: Error: Dart library 'dart:html' is not available on this platform.
import 'dart:html' as html;

```

**Comment by Giuliopretis** [comment]
Same here

**Comment by ponnamkarthik** [comment]
Fixed by merging PR #507
